### PR TITLE
add remote-client diff

### PR DESCRIPTION
**Pull request by baude**
the remote client now can run the diff command to report changes,
modifications, and deletions in an image or container.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: pkg/adapter/runtime_remote.go [hunk 1]
```diff
@@ -831,3 +831,30 @@ func (r *LocalRuntime) Events(c *cliconfig.EventValues) error {
...
+}
+
+func stringToChangeType(change string) archive.ChangeType {
+	switch change {
+	case "A":
+		return archive.ChangeAdd
+	case "D":
+		return archive.ChangeDelete
+	default:
+		if change != "M" {
```
Comment (rhatdan) [inline pkg/adapter/runtime_remote.go]: Why not do 

case "M":
       return archive.ChangeModify
}
logrus.Errorf("'%s' is unknown archive type", change)

Comment (baude) [reply pkg/adapter/runtime_remote.go]: because no value is returned

Comment (rhatdan) [reply pkg/adapter/runtime_remote.go]:
```
	switch change {
	case "A":
		 return archive.ChangeAdd
	case "D":
		return archive.ChangeDelete
	default:
                 logrus.Errorf("'%s' is unknown archive type", change)
                 fallthrough
	case M:
		return archive.ChangeModify
	}
```

Comment (rhatdan) [reply pkg/adapter/runtime_remote.go]: Looks a lot cleaner.

Comment (baude) [reply pkg/adapter/runtime_remote.go]: done

**Comment by rhatdan** [comment]
bot, retest this please

**Comment by rhatdan** [comment]
LGTM
@mheon @jwhonce @vrothberg @TomSweeneyRedHat @giuseppe PTAL

File: pkg/adapter/runtime_remote.go [hunk 2]
```diff
@@ -831,3 +831,30 @@ func (r *LocalRuntime) Events(c *cliconfig.EventValues) error {
...
+func stringToChangeType(change string) archive.ChangeType {
+	switch change {
+	case "A":
+		return archive.ChangeAdd
+	case "D":
+		return archive.ChangeDelete
+	default:
+		logrus.Errorf("'%s' is unknown archive type", change)
+		fallthrough
+	case "B":
```
Comment (vrothberg) [inline pkg/adapter/runtime_remote.go]: I think this should be `"C"`.

**Comment by TomSweeneyRedHat** [comment]
Other than @vrothberg's comment, LGTM assuming happy tests.  They're not at the moment fwiw.

**Comment by jwhonce** [comment]
LGTM

**Comment by rhatdan** [comment]
bot, retest this please
/lgtm

**Pull request merged**